src/include/snapper/dialogs.rb

```diff
@@ -153,24 +147,22 @@ def ModifySnapshotPopup(snapshot)
         )
       end
 
-      cont = VBox(
-        # popup label, %1 is number
-        Label(Builtins.sformat(_("Modify Snapshot %1"), num)),
-        snapshot_term.call("", snapshot)
-      )
-
-      if type == :POST
+      if type != :POST
         cont = VBox(
-          # popup label, %1, %2 are numbers (range)
-          Label(
-            Builtins.sformat(_("Modify Snapshots %1 - %2"), previous_num, num)
-          ),
+          # popup label, %{num} is number
+          Label(_("Modify Snapshot %{num}") % { :num => num }),
+          snapshot_term.call("", snapshot)
+        )
+      else
+        cont = VBox(
+          # popup label, %{pre} and %{post} are numbers
+          Label(_("Modify Snapshot %{pre} and %{post}") % { :pre => pre_num, :post => num }),
           # label
-          Left(Label(Builtins.sformat(_("Pre (%1)"), previous_num))),
+          Left(Label(_("Pre (%{pre})") % { :pre => pre_num })),
```

Comment:
Josef is already waiting to unleash rubocop on the module so these style issues do not matter now.